photon-server/src/test/java/org/photonvision/common/util/LogFileManagementTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2020 Photon Vision.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.photonvision.common.util;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.filefilter.WildcardFileFilter;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.photonvision.common.configuration.ConfigManager;
+import org.photonvision.common.logging.Logger;
+
+public class LogFileManagementTest {
+
+    @Test
+    public void fileCleanupTest() throws IOException {
```

Comment:
Since this test is using the photonvision_config directory within the project, it would be great if, after the unit test finished, the /test directory was deleted.

Reply:
Added removal operations.  Of note: If the test isn't working, the developer will want to single-step debug or comment out the deletion lines to watch what's happening on disk. But, IMO that burden is just fine if you're in troubleshooting mode anyway.